extensionsGallery.json

```diff
@@ -480,7 +480,57 @@
 					],
 					"statistics": [],
 					"flags": "preview"
-				}
+				}, {
+                    "extensionId": "20",
+                    "extensionName": "sqlops-combine-scripts",
+                    "displayName": "Combine Scripts",
+                    "shortDescription": "Create a single combined script from several files.",
+                    "publisher": {
+                      "displayName":"Bateleur IO",
+                      "publisherId": "BateleurIO",
+                      "publisherName":"BateleurIO"
+                    },
+                    "versions": [
+                      {
+                        "version": "1.1.0",
+                        "lastUpdated": "7/13/2018",
+                        "assetUri": "",
+                        "fallbackAssetUri": "fallbackAssetUri",
+                        "files": [
+                        {
+                          "assetType": "Microsoft.SQLOps.DownloadPage",
+                          "source": "https://github.com/BateleurIO/sqlops-combine-scripts/releases/tag/1.1.0"
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.Links.Source",
+                          "source": "https://github.com/BateleurIO/sqlops-combine-scripts"
+                       },
+                       {
+                          "assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+                          "source": "https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/icons/main-icon.png"
+                        },
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.Content.Details",
+                         "source": "https://github.com/BateleurIO/sqlops-combine-scripts/blob/master/README.md"

```

Comment:
change this url to "[https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/README.md](https://raw.githubusercontent.com/BateleurIO/sqlops-combine-scripts/master/README.md)"